website/package.json

```diff
@@ -1,4 +1,15 @@
 {
+  "name": "avodocs",
+  "productName": "Avolites Titan Manual",
+  "description": "Official Manual for the Avolites Titan software",
+  "version": "13.1.0",
```

Comment:
I think I would rather version the app separately rather than following the titan version number.

Reply:
Can change back to the old version number?

Reply:
Changed to 0.1.0 in fc266b4, not sure if this is what you were after